### ISSUE TITLE
Add gradient accumulation

### ISSUE THREAD

**Issue by dmarx** [pytti-tools/pytti-core#61]
suggestion from the Eleuther.AI discord

**Comment by dmarx** [comment]
really simple implementation here: https://towardsdatascience.com/i-am-so-done-with-cuda-out-of-memory-c62f42947dca

I think it also comes out-of-the-box with pytorch-lightning, which could be a good way to transition towards multi-gpu support.

**Comment by dmarx** [comment]
#68
